config/tlm.yaml

```diff
@@ -86,3 +86,5 @@
       value: 0x89ABCDEF
 
 - !include ccsds_header.yaml
+- !include sle_tlm.yaml
```

Comment:
Let's drop this since the file will only be in the DSN repo after the above changes